README.md

```diff
@@ -21,15 +21,21 @@ This image is available on Docker Hub: [lequal/sonarqube](https://hub.docker.com
 
```

Comment:
L12-L16: please remove LEQUAL motivation it is not only LEQUAL needs but a general need to have easyly integrable images.

Reply:
Indeed. That's one of the point of open source.